README.md

```diff
@@ -105,6 +105,7 @@ see https://github.com/sphereio/sphere-jvm-sdk-experimental-java-add-ons
 
 ## Executing integration tests
 
+1. create an API client in the Merchant Center with all available permissions (at the time of writing it is manage_payments manage_my_profile manage_orders view_products view_customers view_payments view_types manage_my_orders manage_types manage_customers manage_products view_orders manage_project)
```

Comment:
@lauraluiz I already updated travis with new credentials, but you need to do this for your local project, too.